tests/e2e/vsphere.go

```diff
@@ -752,3 +808,23 @@ func queryCNSVolumeWithWait(ctx context.Context, client clientset.Interface, vol
 	})
 	return waitErr
 }
+
+// waitForHostToBeInState gets the power state of the host and waits till the desired state is obtained
+func waitForHostToBeInState(ctx context.Context, vs *vSphere, addr string, state string) error {
```

Comment:
you can add a log message after which indicates the target host and target connection state

`waitForHostToBeInState` -> `waitForHostConnectionState`

Reply:
Changed